module_utils/rho_cmd.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2009-2017 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+"""RhoCmd: a base class with common functionality for our Ansible modules."""
+
+import subprocess as sp
```

Comment:
I usually don't like this kind of renaming because it makes the code harder to understand and will require to check from where this is coming from. But I saw that it was present on the previous code so I think is fine to have this here. But we might want to avoid that in the future.